src/admin-views/help-hub/resources/faq_template.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Template for displaying FAQ sections in the help hub.
+ *
+ * @var Tribe__Main $main             The main common object.
+ * @var Hub         $help_hub         The Help Hub class.
+ * @var string      $template_variant The template variant, determining which template to display.
+ * @var array       $section          The current section.
+ */
+
+// Ensure we have valid input data and at least one FAQ.
+if ( empty( $section['faqs'] ) || ! is_array( $section['faqs'] ) ) {
+	return;
+}
+?>
+
+<div class="tec-settings-form__content-section">
+	<div class="tec-settings-form__header-block">
+		<h3 class="tec-settings-form__section-header tec-settings-form__section-header--sub">
+			<?php echo esc_html( $section['title'] ); ?>
+		</h3>
+		<p class="tec-settings-form__section-description">
+			<?php echo esc_html( $section['description'] ); ?>
+		</p>
+	</div>
+	<div class="tec-ui-accordion">
+		<?php
+		foreach ( $section['faqs'] as $faq ) :
+			// Skip invalid FAQ entries.
+			if ( ! is_array( $faq ) || empty( $faq['question'] ) || empty( $faq['answer'] ) ) {
+				continue;
+			}
+
+			// Ensure we have strings for question and answer.
+			$question = is_string( $faq['question'] ) ? $faq['question'] : '';
+			$answer   = is_string( $faq['answer'] ) ? $faq['answer'] : '';
```

Comment:
This logic should happen _way_ before the template - where we build `$section['faqs']` and it does - so is this redundant?

Templates should be dumb. They should assume most of the data is valid.